editor.planx.uk/src/pages/Team/components/AddFlow/CreateFromTemplateFormSection.tsx

```diff
@@ -19,8 +19,10 @@ export const CreateFromTemplateFormSection: React.FC = () => {
   const { values, setFieldValue } = useFormikContext<CreateFlow>();
 
   const { data } = useQuery<{ templates: TemplateOption[] }>(gql`
-    query GetTemplates {
-      templates: flows(where: { is_template: { _eq: true } }) {
+    query GetOnlineSourceTemplates {
+      templates: flows(
+        where: { is_template: { _eq: true }, status: { _eq: online } }
```

Comment:
`online` is not wrapped in double quotes here because it's a graphql-recognized enum value, not string !